client/app/bundles/shared/components/Switch.jsx

```diff
@@ -0,0 +1,35 @@
+// @flow
+import React from 'react';
+import css from './Switch.scss';
+
+type Props = {
+  id?: string
+};
+
+type State = {
+  checked: boolean
+};
+
+export default class Switch extends React.Component<Props, State> {
```

Comment:
It would be great to create a unit test for this component and test toggling it.

Reply:
I've tested this ! Let me know if it is the right way !